protected/controllers/PatientController.php

```diff
@@ -760,9 +760,8 @@ public function actionAdddiagnosis()
 
         $date = $this->processFuzzyDate();
 
-        // If the patient came  from a referral or self registration, then the diagnosis is unconfirmed
-        $is_confirmed = ($patient->patient_source == Patient::PATIENT_SOURCE_REFERRAL
-            || $patient->patient_source == Patient::PATIENT_SOURCE_SELF_REGISTER) ? 0 : null;
+        // Mark diagnosis as unconfirmed, regardless of patient source.
+        $is_confirmed = 0;
```

Comment:
This was used as a parameter to $patient->addDiagnosis(), but now will never change. Perhaps we should change that function to no longer has the $is_confirmed parameter, and juset set is_confirmed in there. Whatcha reckon?